csv.go

```diff
@@ -0,0 +1,180 @@
+package main
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/davecgh/go-spew/spew"
+)
+
+type csvPrinter struct {
+	writer     *csv.Writer
+	file       *os.File
+	filename   string
+	headerDone bool
+}
+
+const (
+	csvTimeFormat = "2006-01-02 15:04:05.000"
+)
+
+func newCSVPrinter(filename string, args []string) (*csvPrinter, error) {
+	file, err := os.Create(filename)
```

Comment:
I would check if the provided file name ends with `.csv`. Otherwise tcping should add it.